request.js

```diff
@@ -1021,22 +1022,20 @@ Request.prototype.readResponseBody = function (response) {
       debug('aborted', self.uri.href)
       // `buffer` is defined in the parent scope and used in a closure it exists for the life of the request.
       // This can lead to leaky behavior if the user retains a reference to the request object.
-      buffer.destroy()
+      buffers = []
```

Comment:
Minor nit: perhaps to be consistent with the changes down below, `bufferLength = 0` should be added after this line as well?